### PR TITLE
chore(#97): adds centos-ci script that deploys images to devshift

### PR DESCRIPTION
**Pull request by MatousJobanek**
* added a new cico script `cico_build_deploy.sh` that deploys images to devshift
* add new targets in `Makefile.deploy.prow` to encapsulate actions that compile binary, build images and push to the registry (without processing the template)
* every image version is tagged - adds the possibility of using `latest` tag
* `latest` tag is used as default one in templates
* fixed dockerfile - added missing environments
* extracted logic that retrieves prow version to separated script so I can use it in the cico script as well

Fixes: #97

### CODE REVIEW

File: cluster/hook-template.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ parameters:
   value: arquillian
 - name: VERSION
   required: true
+  value: latest
```
Comment (dipak-pawar) [inline cluster/hook-template.yaml]: How does the `latest` value help here? What's wrong with particular version?

Comment (bartoszmajsak) [reply cluster/hook-template.yaml]: Reading issue comment can shed some light here. Plus this discussion I believe https://github.com/arquillian/ike-prow-plugins/issues/100#issuecomment-384276306

Comment (dipak-pawar) [reply cluster/hook-template.yaml]: thanks for pointers.

File: cico_setup.sh [hunk 2]
```diff
@@ -30,7 +30,7 @@ function install_deps() {
 }
 
 function run_build() {
-  make build
+  make docker-build
```
Comment (dipak-pawar) [inline cico_setup.sh]: @MatousJobanek I can't see `make docker-build` target .

**Comment by alien-ike** [comment]
Hey @dipak-pawar! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer or pull request approver, but not pull request creator for this command to take an effect.

File: cico_setup.sh (hunk 2 above)
Comment (MatousJobanek) [reply cico_setup.sh]: see here https://github.com/arquillian/ike-prow-plugins/blob/master/.make/Makefile.docker.build#L64-L68

**Comment by MatousJobanek** [comment]
Now, I have realized that I have a bug there - I'm calling `deploy-hook` and `deploy-plugins` outside of the docker container, but there are not installed the go binaries. I'll fix it

**Comment by MatousJobanek** [comment]
@dipak-pawar as I had to merge the changes from master, could you please re-mark it as "ok-without-tests" Thanks

**Comment by MatousJobanek** [comment]
[test]

**Comment by MatousJobanek** [comment]
[test]

**Pull request merged**